### ISSUE TITLE
Support `uv export --script`

### ISSUE THREAD

**Issue by charliermarsh** [astral-sh/uv#8609]
You should be able to point to a PEP 723 script to get a requirements.txt of the resolved dependencies.

**Comment by malikrohail** [comment]
can i work on this? (pull req)

**Comment by zanieb** [comment]
@malikrohail feel free, but note @manzt has some work in progress and said it didn't seem trivial.

**Comment by manzt** [comment]
Probably won't have time to touch for a bit so go right ahead! Thought I'd share WIP in case any of the code is helpful.

**Comment by malikrohail** [comment]
yes that would be helpful and fasten things up, pls share ur progress

**Comment by zanieb** [comment]
(their progress is linked above)

**Comment by malikrohail** [comment]
sounds good, will work on it later today.
